.travis.yml

```diff
@@ -1,3 +1,11 @@
 language: node_js
 node_js:
   - '11.14.0'
+
+script: echo "Running tests against $(node -v)..."
+
+jobs:
+  include:
+    - stage: Produce Coverage
+      node_js: node
+      script: jest --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage
```

Comment:
Could you just pipe output to the tool here, rather than persisting to the filesystem (and having to delete)?

```
jest --coverage --coverageReporters=text-lcov | $(npm bin)/coveralls
```

Reply:
Fixed